package.json

```diff
@@ -59,6 +59,7 @@
     "keymirror": "0.1.1",
     "lodash.clone": "3.0.3",
     "lodash.defaultsdeep": "3.10.0",
+    "lodash.isarray": "^3.0.4",
```

Comment:
Could you remove the caret? We want to keep the versioning locked in so that we can manually update packages and make sure they don't break anything – otherwise packages will update every time we deploy.